polkadot-parachains/src/cli.rs

```diff
@@ -52,69 +52,70 @@ pub enum Subcommand {
 	Revert(sc_cli::RevertCmd),
 
 	/// The custom benchmark subcommmand benchmarking runtime pallets.
-	#[structopt(name = "benchmark", about = "Benchmark runtime pallets.")]
+	#[clap(name = "benchmark", about = "Benchmark runtime pallets.")]
 	Benchmark(frame_benchmarking_cli::BenchmarkCmd),
 
 	/// Try some testing command against a specified runtime state.
 	TryRuntime(try_runtime_cli::TryRuntimeCmd),
 
 	/// Key management CLI utilities
+	#[clap(subcommand)]
```

Comment:
Still don't get why we need this here and not for the others.

Reply:
What is the error when we don't do this? Does it fail at compile time?

Reply:
yeah, it will be fail at compile time